### PR TITLE
fix: headphones carousel example

### PR DESCRIPTION
**Pull request by CarlosSLoureiro**
# Summary

Fix de logic of background color from circle component at the headphones carousel example.

vp-carousel

### Motivation

I've reading the code of the headphones carousel example and I understood that the color of the circle must change due to a code line will be mentioned below in this PR.

## Test Plan

Just try to execute the mentioned example.

## Checklist

- [X] I have tested this on a device and a simulator (yes, in Android and iOS)
- [X] I added the documentation in `README.md` (yes, updated the readme gif example)
- [ ] I updated the typed files (TS and Flow) (No)

### CODE REVIEW

File: example/src/HeadphonesCarouselExample.tsx [hunk 1]
```diff
@@ -64,39 +64,46 @@ const CIRCLE_SIZE = width * 0.6;
...
+    outputRange: [0.2, 0, 0.2],
+  });
 
-        return (
-          <Animated.View
-            key={index}
-            style={[
-              styles.circle,
-              {
-                backgroundColor: color,
```
Comment (CarlosSLoureiro) [inline example/src/HeadphonesCarouselExample.tsx:92]: This it is the mentioned line part above. In addition to the component rendering according to the data length, the color swapping logic does not work correctly.

**Review by troZee — CHANGES_REQUESTED** [review]

**Comment by troZee** [comment]
@CarlosSLoureiro could you resolve conflicts 🙏

**Comment by CarlosSLoureiro** [comment]
@troZee I think it's done.

**Comment by troZee** [comment]
@CarlosSLoureiro I have run a pipeline once again. Once it passed, I will merge it

File: example/package.json [hunk 2]
```diff
@@ -14,10 +14,10 @@
     "@react-navigation/native-stack": "~6.9.12",
     "@react-navigation/stack": "~6.3.16",
     "react": "18.2.0",
-    "react-native": "0.71.8",
+    "react-native": "0.72.3",
```
Comment (troZee) [inline example/package.json:17]: [high]: Could you revert that file?

File: example/ios/Podfile.lock [hunk 3]
```diff
@@ -2,17 +2,16 @@ PODS:
   - boost (1.76.0)
   - CocoaAsyncSocket (7.6.5)
   - DoubleConversion (1.1.6)
-  - FBLazyVector (0.71.8)
```
Comment (troZee) [inline example/ios/Podfile.lock:5]: [high]: Could you revert that file?

**Review by troZee — CHANGES_REQUESTED** [review]

File: example/package.json (hunk 2 above)
Comment (CarlosSLoureiro) [reply example/package.json:17]: When I tried to run the interpolate in the background Color it doesn't worked in 0.71.8 version. That's why I've upgraded the react native version. Maybe it's a specific issue from the current used version in the example...

**Pull request closed without merging**